sdk.nix

```diff
@@ -12,13 +12,16 @@ let
     url = "https://github.com/digital-asset/daml/releases/download/v${version}/daml-sdk-${version}-macos.tar.gz";
     sha256 = sdkSpec.macSha256;
   };
+  extra-args = if version == "2.8.0" then "--install-with-custom-version ${version}" else "";
```

Comment:
these args are now required to avoid network usage on install

Reply:
https://docs.daml.com/2.8.0/getting-started/manual-download.html#install-the-sdk-from-a-tarball
vs
https://docs.daml.com/2.7.6/getting-started/manual-download.html#install-the-sdk-from-a-tarball